src/puppetlabs/trapperkeeper/services/authorization/authorization_service.clj

```diff
@@ -19,7 +20,7 @@
    [this context]
    (let [config (get-in-config [:authorization])
          rules (-> config validate-auth-config! :rules transform-config)]
-     (log/debug "Transformed auth.conf rules:\n" (ks/pprint-to-string rules))
+     (log/debug (trs "Transformed auth.conf rules:") "\n" (ks/pprint-to-string rules))
```

Comment:
Wait, actually doesn't `log/debug` take any number of params and concatenate them? Either way one trs call with everything would read easier